extensions/positron-python/src/client/positron/session.ts

```diff
@@ -605,6 +633,10 @@ export class PythonRuntimeSession implements positron.LanguageRuntimeSession, vs
     }
 }
 
+export function createUniqueId(): string {
+    return Math.floor(Math.random() * 0x100000000).toString(16);
+}
+
```

Comment:
I realize we haven't been consistent on this, but I generally don't like using crypto's randomUUID since it has to pull from the entropy pool (it generates cryptographically secure IDs) so is unnecessarily slow when pseudorandomness is just fine.